ras/packages.py

```diff
@@ -41,15 +41,14 @@ def test(self):
                 'packages_rhel', default=['lshw', 'librtas'])
             self.packages.extend(packages_rhel)
             for package in self.packages:
-                if "anaconda" in process.system_output("yum list installed "
-                                                       "| grep %s | tail -1"
-                                                       % package,
-                                                       shell=True).decode("utf-8"):
+                if process.system_output("yum list installed "
```

Comment:
@shirishaganta Any reason why the following method is not used for all the distros?
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1436/files#diff-75b317494162cabffcfe2bef4575ce52L69

I believe it must work on any given distro.

Reply:
@shirishaganta The above link does not seem to direct properly, I meant `smm.check_installed` here.. https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/ras/packages.py#L68